src/adapters/roxot.js

```diff
@@ -0,0 +1,118 @@
+var CONSTANTS = require('../constants.json');
+var utils = require('../utils.js');
+var events = require('../events.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader');
+
+
+var RoxotAdapter = function RoxotAdapter() {
+  var roxotUrl = "r.rxthdr.com";
+
+  $$PREBID_GLOBAL$$.roxotResponseHandler = roxotResponseHandler;
+  $$PREBID_GLOBAL$$.roxotEvents = events;
```

Comment:
@kir-roxot 
We don't allow exposing internal eventing externally like this. You will need to pass the data through your response handler or use the analytics adapter method to collect data.